### DISCUSSION TITLE
Option for batch / multiple generations ie for Bark?

### DISCUSSION THREAD

**Discussion by luthis1124** [gitmylo/audio-webui#88] [Ideas]
I don't think this would be hard to implement,

Just a checkbox when using Bark to enable batch, and a dropdown or number field to set number of generations

Bark is not always accurate, so the ability to generate like, 10 audios one after another so I can pick the best sounding one would be awesome.

Doesn't need parallelization or anything, just sequential would be fine.

**Comment by gitmylo** [comment]
Refer to #7, it's not possible to have a previously undefined amount of audios in gradio, you can only have one per element, and you cannot add new elements during runtime.
I will be implementing an API as an extension, you could use that, or you could write an extension which is specifically for batch bark generations.
